spec/uploaders/image_uploader_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'carrierwave/test/matchers'
+
+describe ImageUploader do
+  include CarrierWave::Test::Matchers
+  let(:image) { create(:image) }
+  let(:uploader) { ImageUploader.new(image, :file) }
+
+  before do
+    ImageUploader.enable_processing = true
+
+    File.open('app/assets/images/hilbert.jpg') do |f|
+      uploader.store!(f)
+    end
+  end
+
+  after do
+    ImageUploader.enable_processing = false
+    uploader.remove!
+  end
+
+  context 'the thumb version' do
+    it "should scale down a landscape image to be exactly 350 by 350 pixels" do
+      uploader.thumb.should have_dimensions(350, 350)
+    end
+  end
+
+  context 'the small version' do
+    it "should scale down a landscape image to fit within 1680 by 10000 pixels" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.